modules/acid/src/main/ts/ephox/acid/api/colour/RgbaColour.ts

```diff
@@ -7,7 +7,7 @@ const min = Math.min;
 const max = Math.max;
 const round = Math.round;
 
-const rgbRegex = /^rgb\((\d+),\s*(\d+),\s*(\d+)\)/;
+const rgbRegex = /^\s*rgb\s*\(\s*(\d+)\s*,\s*(\d+)\s*,\s*(\d+)\s*\)\s*$/i;
```

Comment:
If you're changing this then we also need to change the RGBA one below as well since they should be almost identical with the exception that rgba takes another value. Also make sure to add some tests similar to your other new ones as well.

- [ ] Fix the RBGA regex and add some tests